### DISCUSSION TITLE
Source of DigitalFilter.h

### DISCUSSION THREAD

**Discussion by nadiDS2** [EmotiBit/EmotiBit_FeatherWing#308] [Q&A] [answered]
In order to understand the digital filters used in [Emotibit Featherwing](https://github.com/EmotiBit/EmotiBit_FeatherWing) and [Emotibit MAX30101](https://github.com/EmotiBit/EmotiBit_MAX30101), I am trying to find the source for the DigitalFilter class used by both.
I found some implementations that have a "DigitalFilter.h" but none of them match the way these two repositories use the filters.

Does anybody know where these filters came from?

**Comment by nitin710** [comment, marked as answer]
Here is the source for the digital filter: https://github.com/EmotiBit/EmotiBit_XPlat_Utils/blob/master/src/DigitalFilter.h

**Reply by nitin710** [reply]
If you have any future questions, please post it on the [EmotiBit forum](https:///www.forum.emotibit.com) as that helps the community grow!
